frontend/src/components/common/Header.tsx

```diff
@@ -79,7 +90,7 @@ const Header = (): JSX.Element => {
                     Home
                   </Link>
                   <Link as={ReactLink} to={Routes.DASHBOARD_PAGE}>
-                    Scheduled Donations
+                    My Scheduled Donations
                   </Link>
                 </>
               )}
```

Comment:
switch order for `User Management` and `View Donations` on Desktop as well (line 99)

Reply:
good catch!